### PR TITLE
CSVS-55 List was replaced to Set where tags are used

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

File: src/main/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultController.scala [hunk 1]
```diff
@@ -124,19 +124,19 @@ class CloudStackVaultController(vaultService: VaultService,
...
 
     if (currentVaultTags.isEmpty) {
-      val newVaultTokenTagKeyList = Set(Tag.Key.VaultRO, Tag.Key.VaultRW).collect {
+      val newVaultTokenTagKeySet = Set(Tag.Key.VaultRO, Tag.Key.VaultRW).collect {
         case tagKey if !currentVaultTags.exists(_.key == tagKey) => createMissingAccountTokenTag(accountId, tagKey)
       }
 
       if (usersIds.nonEmpty) {
-        val newVaultKeyspaceTagKeyList = Set(Tag.Key.VaultHost, Tag.Key.VaultPrefix).collect {
+        val newVaultKeyspaceTagKeySet = Set(Tag.Key.VaultHost, Tag.Key.VaultPrefix).collect {
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultController.scala]: newVaultKeyspaceTags?

File: src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceTestSuite.scala [hunk 2]
```diff
@@ -140,23 +140,20 @@ class CloudStackServiceTestSuite extends FlatSpec with TestData with BaseTestSui
     val value = "value1"
 
     val cloudStackTaskCreator = new CloudStackTaskCreator(cloudStackTaskCreatorSettings) {
-      override def createSetResourceTagsTask(resourceId: UUID, resourceType: Type, tagList: List[Tag]): () => Unit = {
+      override def createSetResourceTagsTask(resourceId: UUID, resourceType: Type, tagSet: Set[Tag]): () => Unit = {
         assert(resourceId == vmId, "resourceId is wrong")
         assert(resourceType == Tag.Type.UserVM, "resourceType is wrong")
-        assert(tagList == List(Tag.createTag(key, value)), "tagList is wrong")
+        assert(tagSet == Set(Tag.createTag(key, value)), "tagSet is wrong")
```
Comment (placccebo) [inline src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceTestSuite.scala]: set of tags is wrong

**Review by placccebo — DISMISSED** [review]

File: src/main/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultController.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/vault/server/controllers/CloudStackVaultController.scala]: yes, it's right

File: src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceTestSuite.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/test/scala/com/bwsw/cloudstack/vault/server/cloudstack/CloudStackServiceTestSuite.scala]: ok

**Review by placccebo — APPROVED** [review]

**Pull request merged**